src/components/Lend/LoanSearch/LoanSearchGenderFilter.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<fieldset class="tw-flex tw-flex-col tw-gap-2 tw-my-2">
+		<kv-radio
+			v-for="option in genderOptions"
+			:key="option.key"
+			:value="option.key"
+			v-model="selectedGender"
+			@change="setGender"
+		>
+			{{ option.title }}
+		</kv-radio>
+	</fieldset>
+</template>
+
+<script>
+import KvRadio from '~/@kiva/kv-components/vue/KvRadio';
+
+export const BOTH_KEY = 'both';
+export const BOTH_TITLE = 'All genders';
+export const FEMALE_KEY = 'female';
+export const FEMALE_TITLE = 'Women';
+export const MALE_KEY = 'male';
+export const MALE_TITLE = 'Men';
+const GENDER_KEYS = [BOTH_KEY, FEMALE_KEY, MALE_KEY];
+
+export default {
+	components: {
+		KvRadio,
+	},
+	props: {
+		gender: {
+			type: String,
+			// TODO: determine if the default should be "null" (old component behavior) or "both"
+			default: BOTH_KEY
+		}
+	},
+	data() {
+		return {
+			selectedGender: this.gender || BOTH_KEY,
+			genderOptions: [
```

Comment:
The Gender options will likely become a dynamic data set passed in at some point but in this limited case the data set is currently small enough that it's ok to have them static and hard-coded here.

Reply:
Yeah I was wondering if this component should be a generic radio group, but it'll be easy to adjust later.